lib/mixlib/install/backend/artifactory.rb

```diff
@@ -54,24 +56,31 @@ def info
           end
         end
 
-        private
+        def product_versions
+          params = {
+            g: "com.getchef",
+            a: options.product_name,
+            repos: ARTIFACTORY_REPOSITORY
+          }
+          results = client.get("/api/search/versions", params)["results"]
```

Comment:
In this particular case using group id and artifact id will likely give us some mileage.  I chose this because it's an anonymous request.  However, now that we are introducing additional Artifactory queries it's time to implement the AQL post and along with authentication.  It would also improve error handing up front when using the unstable channel.

Are you okay with allowing this to merge, and I'll start a new PR immediately to refactor all AQL stuff?